addons/miscFixes/patchCUP/config.cpp

```diff
@@ -56,6 +56,52 @@ class CfgVehicles {
             };
         };
     };
+    // Tweaks to the GTK Boxer's handling (accel/braking) + HMG swap to M3M + countermeasures move to gunner
+    class Wheeled_APC_F: Car_F {
+        class NewTurret;
+        class Turrets {
+            class MainTurret: NewTurret {
+                class Turrets {};
+            };
+        };
+    };
+    class CUP_Boxer_Base: Wheeled_APC_F {};
```

Comment:
Not writing out the full sub classes in this class results in the classes in CUP_Boxer_Bas_HMG inheriting from the versions from Wheeled_APC_F. Luckily this time it doesn't cause issues since the Turrets class isn't changed in CUP_Boxer_Base, but it should be fixed to avoid any future issues.

@PabstMirror lmk if I got something wrong above, but that's been my understanding

Reply:
I think it's ok as is, at least we don't seem to be getting any ubc

Reply:
I thought that would be inheriting the previous class turret configs (`Wheeled_APC_F`), so if any other mod changes the `CUP_Boxer_Base`'s `turrets` class , we might override those changes even if we didn't change those specific parameters since we're changing the turret inheritance? Unless I misread the inheritance of the turret.

Reply:
CUP_Boxer_Base doesn't touch `turrets` as far as I can tell, only the hmg/gmg variants do

Reply:
Right, but if we load another mod that touches it `turrets` in CUP_Boxer_Base, wouldn't any change it makes be lost?

Reply:
the sub-classes fully redefine `turrets`
so any change to `CUP_Boxer_Base` would just be ignored

```
class CUP_Boxer_Base_HMG: CUP_Boxer_Base {
		class Turrets {
			class MainTurret: NewTurret { ... }
```

Reply:
Sounds good, ty.